### ISSUE TITLE
Function calls report incorrect return values

### ISSUE THREAD

**Issue by Sharparam** [varren/ViragDevTool#8]
Steps:

1. Add `GetChatTypeIndex`
2. Set arguments to `"RAID"`
3. Press the function in list to execute
4. Observe return value

Expected: `4` (as reported by `/dump GetChatTypeIndex("RAID")`)

Actual: `0`

Screenshots:
![Log screenshot](https://rfl.im/JcE)
![Chat screenshot with /dump](https://rfl.im/JcS)

**Comment by varren** [comment]
Yeah, ty for reporting. It's just a problem with not intuitive/not standard and badly documented arguments console.

You don't have to use quotations for strings right now:
https://i.gyazo.com/fc8cc68727d5fa1819f540bdcbfd5948.png

`"RAID"` should be just `RAID` in console

Will have to update this when i have time and figure out some standard way to use this.
Cant fix right now.

**Comment by Sharparam** [comment]
Ah, I see. Quite confusing given how it's formatted.
